### ISSUE TITLE
[Bug] Unable to save Font Settings

### ISSUE THREAD

**Issue by EinWaterBottle** [mclemente/fvtt-module-polyglot#343]
**Describe the bug**
Polyglot Font Settings are not saving, specifically the "Logographical" and "Only Alphabetic" checkboxes. "Font Size" at least seems to be able to be altered without issue.

**To Reproduce**
1. Open Font Settings under Polyglot module settings.
2. Check or uncheck "Logographical" or "Only Alphabetic" checkboxes.
3. Save settings
4. Reload page as instructed by Foundry
5. See error.

**Expected behavior**
I expect the settings to be altered and stay altered.

**Screenshots**
![chrome_CfKT3xQWGv](https://github.com/mclemente/fvtt-module-polyglot/assets/106705473/f5a1a718-5595-4632-8863-d2ca2f097879)

 - Game System and version: dnd5e 2.4.1
 - Polyglot Version: 2.3.18
 - I am not using Babele.

**Additional context**
I believe the "Logographical" and "Only Alphabetic" checkboxes actually clear themselves rather than not being changed. 

In testing, I deactivated every module to find possible incompatibilities. The issue remains even with only this module active.

**Comment by mclemente** [comment]
I've tried to reproduce this on a new world and couldn't get to reproduce it, the checkboxes saved correctly. Could you try this on a new world?

Do you have any additional fonts? How are you hosting the game?

**Comment by EinWaterBottle** [comment]
I run servers via Forge, and run only the fonts Polyglot and core Foundry provide.

1. I created a new world as you said and tried reproducing the same issue. This time, however, it saved. So it's likely the default settings work.
2. Then, I messed around with some of the General Settings. I activated "Add Foundry's fonts of Font Settings" and "Add Polyglot's fonts to Core", restarted, and found the Font Settings checkboxes had all cleared. 
3. Next, I deactivated those two settings and messed around with Font Settings checkboxes, which were applying correctly.
4. This time, I activated only "Add Foundry's fonts of Font Settings". The checkboxes were applying correctly.
5. Then I activated only "Add Polyglot's fonts to Core". The checkboxes were applying correctly too.
6. I activated both. The checkboxes were now clearing like before.

So I assume from my testing it's those two settings being active together that's causing the issue. Which is an easy solution for me, since I'll just toggle off one of those settings in my home server, but it's still something that should be brought to your attention.
